detector/calorimeter/SEcal05_Helpers.h

```diff
@@ -83,7 +88,7 @@ class SEcal05_Helpers {
   SEcal05_Helpers();
 
   ~SEcal05_Helpers() {
-    delete _layering;
+    if ( _layering ) delete _layering; _layering=NULL;
```

Comment:
This check was not necessary.
`delete NULL` is a no-op. Also setting the variable in the destructor is not necessary.
These changes introduce a warning about "misleading indentation as well.